test/functional/rpc_psbt.py

```diff
@@ -985,6 +1033,5 @@ def test_psbt_input_keys(psbt_input, keys):
         self.log.info("Test descriptorprocesspsbt raises if an invalid sighashtype is passed")
         assert_raises_rpc_error(-8, "'all' is not a valid sighash parameter.", self.nodes[2].descriptorprocesspsbt, psbt, [descriptor], sighashtype="all")
 
-
```

Comment:
`PEP 8: E305 expected 2 blank lines after class or function definition, found 1`. Let's not remove this line.

Reply:
added the line back, thanks